backend/src/db/migrations/0000_dry_silver_centurion.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS "transfer_usdc" (
```

Comment:
the names are weird is that expected to be pushed ?

Reply:
Yep, they're the migrations to be applied to the database.